ext/TrixiConvexECOSExt.jl

```diff
@@ -0,0 +1,144 @@
+# Package extension for adding Convex-based features to Trixi.jl
+module TrixiConvexECOSExt
+
+# Required for coefficient optimization in P-ERK scheme integrators
+if isdefined(Base, :get_extension)
+    using Convex
+    using ECOS
+else
+    # Until Julia v1.9 is the minimum required version for Trixi.jl, we still support Requires.jl
+    using ..Convex
+    using ..ECOS
+end
+
+# Use other necessary libraries
+using LinearAlgebra: eigvals
+const MOI = Convex.MOI
+
+# Use all exported symbols to avoid having to rewrite `recipes_makie.jl`
+using Trixi
+
+# Use additional symbols that are not exported
+using Trixi: PolynomialOptimizer
+
+# Import functions such that they can be extended with new methods
+import .PolynomialOptimizer: filter_eigvals, bisection, undo_normalization!
+
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+function stability_polynomials(cons_order, num_stage_evals, num_eig_vals,
+                               normalized_powered_eigvals_scaled, pnoms,
+                               gamma::Variable)
+    # Initialize with zero'th order (z^0) coefficient
+    for i in 1:num_eig_vals
+        pnoms[i] = 1.0
+    end
+
+    # First `cons_order` terms of the exponential
+    for k in 1:cons_order
+        for i in 1:num_eig_vals
+            pnoms[i] += normalized_powered_eigvals_scaled[i, k]
+        end
+    end
```

Comment:
Can you try changing the order of the loops and check if that makes any difference in performance? 
As `k` is the inner variable, we should loop over this also further "inside".

Reply:
Oh, I forgot that Julia is column-major. My bad, then the current version should indeed be faster!